app/page/authorization/userOperation.tsx

```diff
@@ -124,11 +126,18 @@ function UserOperationConfirmation(props: { userOpLog: UserOperationLog }) {
   const markUserOperationRejected = useStorage(
     (storage) => storage.markUserOperationRejected
   )
-  const rejectUserOperation = async () => {
+  const rejectUserOperation = useCallback(async () => {
     markUserOperationRejected(userOpLog.id)
     navigate(Path.Index)
+    if (!popupOpened) {
+      // Close this window asynchronously
+      window.onbeforeunload = null
+      setTimeout(() => {
+        window.close()
+      }, 100) // After 0.1 seconds
+    }
```

Comment:
This doesn't make sense to me. When this function is called, the popup must be opened. Does this really solves your problem?

Reply:
No, this PR still has a pending bug to handle.
The main purpose of this PR is to resolve the handling of the [newly created window](https://github.com/pilagod/waallet-extension/blob/3d96d44ac7e1ddc4c75bb743fefc13336b6d25e8/background/index.ts#L75-L81) when the user clicks the `Reject` button: it should be closed instead of displaying the `Info` page.

However, there's a bug here where even when the `Popup Script` is opened, `popupOpened` always remains `false`.
This causes the entire `Popup Script` to close when the user clicks the `Reject` button in the `Popup Script` (rather than in the `newly created window`).